src/main/editDelete/preProcess.ts

```diff
@@ -50,6 +31,74 @@ const injectAttributes: MapCallback<PartialWord, Word> = (word, i) => ({
   fileName: 'PLACEHOLDER FILENAME',
 });
 
+const constructWord = (
+  word: string,
+  startTime: number,
+  duration: number,
+  outputStartTime: number,
+  key: string
+) => {
+  const newWord: Word = {
```

Comment:
* Inline this and directly return the object
* Give the function a type definition including a return type
* Can make the function inline as well. See example below

`const returnAnObject: (something: string) => { putHere: string; } = (something) => ({ putHere: something });`